libfreerdp/core/settings.c

```diff
@@ -460,8 +460,7 @@ rdpSettings* freerdp_settings_new(DWORD flags)
 		goto out_fail;
 
 	if (!freerdp_settings_set_uint32(settings, FreeRDP_MultitransportFlags,
-	                                 (TRANSPORT_TYPE_UDP_FECR | TRANSPORT_TYPE_UDP_FECL |
-	                                  TRANSPORT_TYPE_UDP_PREFERRED | SOFTSYNC_TCP_TO_UDP)))
+	                                 TRANSPORT_TYPE_UDP_FECR))
```

Comment:
Actually, this flag is for now not necessary too. Or will we run into issues, when we have no flags set here?

Reply:
@pnowack if no flags are set multitransport is automatically disabled. (we need to announce something, we can always tell the server it does not work later on)

Reply:
> we need to announce something, we can always tell the server it does not work later on

alright.